### PR TITLE
Included cleanup check in hyperconverged test-suite

### PR DESCRIPTION
**Pull request by Swarnalatha-K**
Signed-off-by: swarna <swarnalatha@cloudbyte.com>

**What this PR does / why we need it**:
- Enhance ansible playbooks to deploy applications on their own namespace and set flag for cleanup result.(Hyperconverged test-suite)
- Stern logger moved to prereq.yml
- Runs each application on its own namespace
- Performs apply/delete applications using modularization by passing required arguments.
-Cleanup gives out result by setting flag"Cleanup Pass" or "Cleanup Fail" 

 **Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:
@ksatchit Updated the changes for percona-diff-NS and drain-node and scaleup-cassandra-pods playbooks. Can you please review it.

### CODE REVIEW

**Comment by Swarnalatha-K** [comment]
[cassandra-scaleup.txt](https://github.com/openebs/openebs/files/2022615/cassandra-scaleup.txt)
**Ansile-lint output:**
```
devops@devops-virtual-machine:~/openebs/e2e/ansible/playbooks/hyperconverged/test-scaleup-cassandra-pods$ ansible-lint test-scaleup-cassandra-pods.yml
devops@devops-virtual-machine:~/openebs/e2e/ansible/playbooks/hyperconverged/test-scaleup-cassandra-pods$
```

File: e2e/ansible/playbooks/hyperconverged/percona-openebs-ownnamespace/openebs-percona-diffns-pod.yml [hunk 1]
```diff
@@ -174,31 +112,45 @@
...
          failed_when: "result.rc != 0"
 
-       - name: Terminate the log aggregator
-         shell: source ~/.profile; killall stern
-         args:
-           executable: /bin/bash
-         delegate_to: "{{groups['kubernetes-kubemasters'].0}}"
 
-       - name: Set flag tp pass if test case is passed
+       - name: Test playbook passed
```
Comment (dargasudarshan) [inline e2e/ansible/playbooks/hyperconverged/percona-openebs-ownnamespace/openebs-percona-diffns-pod.yml]: Set Flag as "Test Passed"

File: e2e/ansible/playbooks/hyperconverged/percona-openebs-ownnamespace/openebs-percona-diffns-pod.yml [hunk 2]
```diff
@@ -174,31 +112,45 @@
...
 
      always:
+       - block:
+
+           - include: k8s-percona-openebs-pod-cleanup.yml
+             when: clean | bool
+
+           - name: Test Cleanup Passed
+             set_fact:
+               cflag: cleanup_pass
```
Comment (dargasudarshan) [inline e2e/ansible/playbooks/hyperconverged/percona-openebs-ownnamespace/openebs-percona-diffns-pod.yml]: Set Flag as "Cleanup Passed"

**Comment by Swarnalatha-K** [comment]
@dargasudarshan Implemented the review comments, Please review it.

**Pull request merged**